tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareReceivedInMultipleWays.feature

```diff
@@ -309,7 +309,7 @@ Feature: share resources where the sharee receives the share in multiple ways
       | permissionsRole | Editor        |
     And user "Brian" should be able to create folder "/Shares/child1/fo1"
     And user "Brian" should be able to create folder "/Shares/child1/child2/fo2"
-    But user "Brian" should not be able to create folder "/Shares/parent/fo3"
+    And user "Brian" should not be able to create folder "/Shares/parent/fo3"
```

Comment:
This scenario needs `Then` at line 310, and `But` can stay here.

It would also be nice to have a rule that checks for Scenarios that do not have any `When` step - this scenario needs to be looked at, and work out how/where to put the `When` keyword.

Reply:
I raised issue https://github.com/gherlint/gherlint/issues/122 to check for no `When` step.

Reply:
@phil-davis Let's see what @saw-jan has to say about it.

Reply:
@phil-davis For now, should I update I manually and push it?

Reply:
IMO, let's keep the scenarios as it is for now. (Given...Then)
We will implement more rules to gherlint in the future and then the updated linter should catch this scenario.

Reply:
Sure, please review the changes again.